packages/paste-website/src/pages/components/combobox/index.mdx

```diff
@@ -199,7 +246,7 @@ In the example below the value of the Combobox is stored in a piece of our appli
   {controlledComboboxExample}
 </LivePreview>
 
-#### useCombobox state hook
+### useCombobox state hook
 
 <Callout variant="warning" marginY="space70">
   <CalloutHeading as="h5">Power user move!</CalloutHeading>
```

Comment:
nitpick: There's a sentence hidden here that says "An example usecase..." where it should be "An example use case..."